navigation/BottomTabNavigation.jsx

```diff
@@ -120,3 +120,4 @@ export default function BottomTabNavigation() {
     </Tab.Navigator>
   );
 }
```

Comment:
for this component you send the same props you can do it this way 

<SupportIcon
     width={horizontalScale(25)}
     height={verticalScale(25)}
     fill={focused ? theme.colors.primary_green : theme.colors.cool_gray }
   />
   
   const getAppropriateProps = (value) => ({
           width: horizontalScale(25),
           height: verticalScale(25),
           fill: focused ? theme.colors.primary_green : theme.colors.cool_gray 
   })

and then 

tabBarIcon: ({focused}) => ( <SupportIcon {...getAppropriateProps(focused)} />

Reply:
Done ! Thank you .